ubiquibot-config-default.yml

```
@@ -0,0 +1,41 @@
+---
+chain-id: 1
+base-multiplier: 1000
+issue-creator-multiplier: 2000
+time-labels:
+  - name: "Time: <1 Hour"
+    weight: 0.125
+    value: 3600
+  - name: "Time: <1 Day"
+    weight: 1
+    value: 86400
+  - name: "Time: <1 Week"
+    weight: 2
+    value: 604800
+  - name: "Time: <2 Weeks"
+    weight: 3
+    value: 1209600
+  - name: "Time: <1 Month"
+    weight: 4
+    value: 2592000
+priority-labels:
+  - name: "Priority: 0 (Normal)"
+    weight: 1
+  - name: "Priority: 1 (Medium)"
+    weight: 2
+  - name: "Priority: 2 (High)"
+    weight: 3
+  - name: "Priority: 3 (Urgent)"
+    weight: 4
+  - name: "Priority: 4 (Emergency)"
+    weight: 5
+auto-pay-mode: true
+analytics-mode: true
+incentive-mode: false
+max-concurrent-bounties: 2
+comment-element-pricing:
+  text: 0.1
+  link: 0.5
+  list: 0.5
+  code: 5
+  image: 5
```

To reduce ambiguity what if we figured a way to directly refer to the markdown or HTML syntax? These aren't great but its a proposal I hope that we can improve:

Markdown version

```suggestion
  "": 0.1
  "[]()": 0.5
  "- ": 0.5
  "``````": 5
  "<img": 5
```

HTML version

```suggestion
  "": 0.1
  "a": 0.5
  "li": 0.5
  "code": 5
  "img": 5
```

Regex might be less clear.

---

[mdast](https://github.com/syntax-tree/mdast-util-from-markdown) already does a great job on parsing markdown files. We just need to tweak it for our needs. In the end all github comments are markdown files initially. So converting md to html and then parsing that html will be less precise and error prone compared to parsing an initial md comment.

---

This is not meant to actually change any behavior under the hood. It is only used as an unambiguous way to refer to the elements. If you try representing the config property names as markdown it looks like a mess.